src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,123 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static class Node<K, V> {
```

Comment:
Don't use getters and setters in the class Node since we have access to private fields of the inner class, so using getters or setters is redundant.